### PR TITLE
Github Actions

### PR DESCRIPTION
**Pull request by dplewis**

### CODE REVIEW

**Comment by dplewis** [comment]
@davimacedo How does this look?

File: .github/workflows/pages.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: docs
+on:
```
Comment (davimacedo) [inline .github/workflows/pages.yml:2]: I am not sure because it is new for me, but I'd try with:
on:
  release:
    types: [published]

**Review by davimacedo — COMMENTED** [review]
@dplewis great job! Just left a comment.

File: .github/workflows/pages.yml (hunk 1 above)
Comment (dplewis) [reply .github/workflows/pages.yml:2]: I can change it when we do a release. I know it works but still want to test it on push

Comment (davimacedo) [reply .github/workflows/pages.yml:2]: Makes sense

**Review by davimacedo — APPROVED** [review]
LGTM!

**Comment by davimacedo** [comment]
I will try to remove Travis from being required

**Comment by dplewis** [comment]
Thanks, I didn’t know the best way to remove it

**Comment by davimacedo** [comment]
Found it. We should be good now.

**Comment by davimacedo** [comment]
For reference: Settings -> Branches -> Branches Protection Rules > Edit Master

There you will find a list of checkboxes with all integrations for you to check which ones are required.

**Comment by dplewis** [comment]
Thanks, the docs work

https://github.com/parse-community/Parse-SDK-JS/tree/gh-pages

**Pull request merged**